src/reconcile/rec_write.c

```diff
@@ -2132,6 +2132,8 @@ __wti_rec_pack_delta_internal(
     *head_byte = flags;
 
     ++header->u.entries;
+    WT_STAT_CONN_DSRC_INCR(session, rec_page_delta_internal);
```

Comment:
Should we increase these stats when we actually write them in __rec_split_write?